R/sig-extremes.R

```diff
@@ -7,7 +7,7 @@
 #'@importFrom rmarkdown render
 #'@examples
 #'library(jsonlite)
-#'data <- fromJSON(system.file('extdata','extremes','extremes-example.json',package = 'repgen'))
```

Comment:
you could leave the original file (unless it's no longer a valid dataset to test with) and just have two examples here?